python/cuml/dask/common/dask_arr_utils.py

```diff
@@ -182,7 +181,12 @@ def to_sp_dask_array(cudf_or_array, client=None):
         return cudf_or_array
 
     else:
-        if scipy.sparse.isspmatrix(cudf_or_array):
+        if has_scipy():
+            from scipy.sparse import isspmatrix as scipy_sparse_isspmatrix
+        else:
+            def scipy_sparse_isspmatrix(x):
+                return False
```

Comment:
Maybe instead:

`if has_scipy() and scipy.sparse.isspmatrix()` (and rely on short circuit evaluation) rather than implementing the dummy function? I thought the dummy function was a little more complex

Reply:
That wouldn't work, because you'd first need to import `scipy.sparse` before using `scipy.sparse.matrix`.

Reply:
Actually, #2018 removed this section, so our discussion is moot. The file `python/cuml/dask/common/dask_arr_utils.py` will remain unmodified by this PR.